test/extra_storage.jl

```diff
@@ -57,3 +59,103 @@ for iter in 1:10
     )
     @test tlmo.counter < lmo_calls0
 end
+
+
+### Away Frank-Wolfe
+
```

Comment:
Add the following in a `@testset` block, that gives it a name and we understand what is going on

Reply:
This has a testset in `runtests.jl`. It's something like 
```
@testset "Extra vertex storage" begin
include("extra_storage.jl")
end
```